pro/scripts/customRequestv1.ts

```diff
@@ -0,0 +1,349 @@
+// overwrites request.ts to handle redirection to maintenance
+
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import { ApiError } from './ApiError'
+import type { ApiRequestOptions } from './ApiRequestOptions'
+import type { ApiResult } from './ApiResult'
+import { CancelablePromise } from './CancelablePromise'
+import type { OnCancel } from './CancelablePromise'
+import type { OpenAPIConfig } from './OpenAPI'
+import { URL_FOR_MAINTENANCE } from 'utils/config'
+
+const isDefined = <T>(
```

Comment:
dommage de ne pas pouvoir tracer les changements par rapport à l'original.
idées : 
* 1er commit: create customRequestv1, avec un fichier is v1/core/request.ts
* 2em commit: add maintenance redirect to customRequestv1, avec les changements